src/app.js

```diff
@@ -149,6 +149,23 @@ const App = Marionette.Application.extend({
     return this;
   },
 
+
+  /**
+   * Sets the app lifecycle to not running
+   * then sets the app lifecycle to running with ending state
+   *
+   * @public
+   * @method restart
+   * @memberOf App
+   * @param {Object} [options] - Settings for the App passed through to events
```

Comment:
there's no options or event in this function.
Also these docs says it returns the app.. it doesn't but it should.